dev/ci/docker/README.md

```diff
@@ -4,31 +4,27 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-The Gitlab CI will build a docker image unless the CI environment variable
+The Gitlab CI will build a Docker image unless the CI environment variable
 `SKIP_DOCKER` is set to `true`. This image will be
 stored in the [Gitlab container registry](https://gitlab.com/coq/coq/container_registry)
 under the name given by the `CACHEKEY` variable from
 the [Gitlab CI configuration file](../../../.gitlab-ci.yml).
 
-In Coq's default CI, `SKIP_DOCKER` is set so as to avoid running a lengthy redundant job.
+`SKIP_DOCKER` defaults to "true" to avoid running a lengthy redundant job.
```

Comment:
The previous phrasing was maybe a bit unclear but the new phrasing is incorrect since the default is actually to not skip the job if the variable is unset (and it is only set in https://gitlab.com/coq/coq but not by default on people's forks).

Reply:
Where is the variable set in gitlab.com?  I gather it's not part of the source tree.  We ought to document how to set that in a fork so others won't wastefully run the docker step every time.  Right?  (Though users may not suspect they have to do anything like that.)

FWIW, a BUILD_DOCKER variable could be set to `CI_PROJECT_PATH=coq/coq` in Gitlab and then the yml could check that value.  If it matches, don't build.  If it's "true", run.  If it's not set, run.

Just curious, is it possible for the fork to access the Docker cache from coq/coq master?

Reply:
It's set in the "Variables" section at in the CI/CD settings on a GitLab repository. It's a bit wasteful to run this job every time but it won't rebuild the Docker image either if it's already available, just wait a few minutes loading a VM for nothing. It's a tradeoff if users want to bother manually rebuilding the Docker image every time this is needed or run this job by default.

>Just curious, is it possible for the fork to access the Docker cache from coq/coq master?

Yes, it should be possible, but it currently isn't set up that way. (Maybe we should change this. I don't know.)

Reply:
Here?  I don't see any "Variables" section on the pages in this submenu.

![image](https://user-images.githubusercontent.com/1253341/103161200-17cce280-4793-11eb-8783-466b496f3ac5.png)

Reply:
Not here. In the setting menu. You probably can't see it because it's only shown to admins.